src/components/AvatarCropModal/components/ImageCropView.js

```diff
@@ -0,0 +1,61 @@
+import PropTypes from 'prop-types';
+import React, {useMemo} from 'react';
+import {Image, View} from 'react-native';
+import {PanGestureHandler} from 'react-native-gesture-handler';
+import Animated from 'react-native-reanimated';
+import styles from '../../../styles/styles';
+import Icon from '../../Icon';
+import * as Expensicons from '../../Icon/Expensicons';
+
+const AnimatedImage = Animated.createAnimatedComponent(Image);
+
+const propTypes = {
+    /** Link to image for cropping   */
+    imageUri: PropTypes.string,
+
+    /** Size of the image container that will be rendered */
+    containerSize: PropTypes.number,
+
+    /** Styles for image component */
+    imageStyle: PropTypes.arrayOf(PropTypes.object),
+
+    /** Callback to execute when the Image lays out */
+    onLayout: PropTypes.func,
+
+    /** Callback to execute when user panning image */
+    // eslint-disable-next-line react/forbid-prop-types
+    panGestureEventHandler: PropTypes.object,
+};
+
+const defaultProps = {
+    imageUri: '',
+    containerSize: 0,
+    imageStyle: null,
+    onLayout: () => { },
+    panGestureEventHandler: () => { },
+};
+
+const ImageCropView = (props) => {
+    const containerStyles = useMemo(() => ({
+        height: props.containerSize,
+        width: props.containerSize,
+    }), [props.containerSize]);
```

Comment:
Move this to styleutils or try to use existing functions?

Reply:
Existing functions? What do you mean? 
Also, could you please write an example of how I can move the dynamic variable to StyleSheet?

Reply:
You can use a normal function that returns a style object. Then use this in style prop.